04/app.js

```diff
@@ -6,4 +6,41 @@ const user = {
         month: '04',
         year: '1985'
     }
-}
\ No newline at end of file
+}
+
+const day = new Date().getDate();
+const month = new Date().getMonth() + 1;
+const year = new Date().getFullYear();
+
+
+function isTodayYourBirthday() {
```

Comment:
Ta funkcje jest dobrze napisana, ale jeszcze lepiej by było jakby przyjmowała parametr, który przechowuje dane dot. urodzenia. Dodatkowo informacje o aktualnej dacie też warto tam zawrzeć.

Wtedy ta funkcja jest samo wystarczalna, a dokładniej to ujmując niezależna od zewnętrznych elementów.
Obecnie zmiana nazwy zmiennej z poza tej funkcji spowoduje błąd podczas jej uruchamiania.

Pamiętaj o tej zależności :)